docs/mvc/security/cors-policy/sample/project.json

```diff
@@ -0,0 +1,34 @@
+﻿{
+  "webroot": "wwwroot",
+  "version": "1.0.0-*",
+
+  "dependencies": {
+    "Microsoft.AspNet.Server.IIS": "1.0.0-beta4",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta4",
+    "Microsoft.AspNet.Mvc": "6.0.0-beta4",
+    // (Other dependencies not shown)
```

Comment:
I think support for comments in the project.json files was removed. I would recommend removing that.